server/util/networking/networking.go

```diff
@@ -761,8 +761,11 @@ func setupVethPair(ctx context.Context, netns *Namespace) (_ *vethPair, err erro
 		{"FORWARD", "-i", device, "-o", vp.hostDevice, "-j", "ACCEPT"},
 	}...)
 
-	for _, rule := range iptablesRules {
-		if err := runCommand(ctx, append([]string{"iptables", "--wait", "-A"}, rule...)...); err != nil {
+	// Insert elements in reverse order to the top of the table, to preserve
+	// the current ordering of the rules in the slice.
```

Comment:
Could you also add a comment about why the rules are going at the top of the table instead of appended at the end? (did we have some other iptables rule that was overriding these?)